emails/tests/fixtures/domain_recipient_expected.email

```diff
@@ -22,127 +22,157 @@ should work
 
 Sent with [ProtonMail](https://protonmail.com/) Secure Email.
 --==[BOUNDARY1]==
-Content-Type: text/html; charset="utf-8"
+Content-Type: text/html; charset="us-ascii"
 MIME-Version: 1.0
-Content-Transfer-Encoding: base64
+Content-Transfer-Encoding: 7bit
```

Comment:
question (blocking): Doesn't AWS SES (or even the sending MTA?) control the `Content-Type` and `Content-Transfer-Encoding` ? So some of our fixtures should still use `utf-8` `base64` right? To make sure we're still handling those ?

Reply:
[RFC 5598](https://datatracker.ietf.org/doc/html/rfc5598#section-4.3) calls the first server the Mail Submission Agent (MSA). It verifies the message is formatted correctly, and then sends it on it's way. It can change the form of the data, like the `Content-Transfer-Encoding`, but it does not have to.

I agree, I'll add a non-English test so that we'll have a UTF-8 example.